Products/PleiadesEntity/browser/adapters/__init__.py

```diff
@@ -246,7 +247,13 @@ def get(self):
         if prefix and isinstance(value, str):
```

Comment:
I don't understand how this change is going to alter the value retrieved by `self.context[key]` in `get()`. I'm probably missing something though, since I don't really understand where the `__get__keysuffix` stuff even comes into play. Maybe we can find a way to make this code more scrutable, perhaps using a descriptor class?